### DISCUSSION TITLE
add Renovate configuration file in these location `.gitea/renovate.json`

### DISCUSSION THREAD

**Discussion by pabl-o-ce** [renovatebot/renovate#23386] [Suggest an Idea]
### Tell us more.

Hi guys based in this url [docs](https://docs.renovatebot.com/configuration-options/)

is possible to add this location:
`.gitea/renovate.json`

got get a better `gitea` flavor in organizing this files in the repo

**Comment by viceice** [comment]
we should better support `.{platform}/renovate.json` so we don't try gitea folder on other platforms and vice versa.

**Reply by jamietanna** [reply]
Raised https://github.com/renovatebot/renovate/issues/40049 for this

**Comment by pabl-o-ce** [comment]
yeah totally! the idea is that :)

Also is possible to add a way to let renovate bot find .drone.yml that is in the same `.{platform}/.drone.yml`

**Comment by jamietanna** [comment]
Raised https://github.com/renovatebot/renovate/issues/40050 for this, thanks!
